packages/react-devtools-shared/src/devtools/views/Components/SelectedElement.js

```diff
@@ -415,19 +426,27 @@ function InspectedElementView({
 
         <NativeStyleEditor />
 
-        {ownerID === null && owners !== null && owners.length > 0 && (
+        {showRenderedBy && (
           <div className={styles.Owners}>
             <div className={styles.OwnersHeader}>rendered by</div>
-            {owners.map(owner => (
-              <OwnerView
-                key={owner.id}
-                displayName={owner.displayName || 'Anonymous'}
-                hocDisplayNames={owner.hocDisplayNames}
-                id={owner.id}
-                isInStore={store.containsElement(owner.id)}
-                type={owner.type}
-              />
-            ))}
+            {owners !== null &&
+              owners.length > 0 &&
```

Comment:
Are you intentionally checing this condition twice (also in `showRenderedBy`)

Reply:
Yes for Flow. I guess I could technically get rid of the second `owners.length` check if I moved `owners !== null && owners.length > 0` above into a second variable but it seemed kind of not worth doing.

Reply:
I'll just add the Flow casting to get rid of the second check though :smile: I don't mind.